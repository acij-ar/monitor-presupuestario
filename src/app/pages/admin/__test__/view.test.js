jest.mock('../components/datasets-form', () => () => <div id="datasets-form" />);

const React = require('react');
const { shallow } = require('enzyme');

const Admin = require('../view');

describe('Admin page', () => {
  it('should match snapshot', () => {
    const props = {
      pageName: 'admin',
    };
    const wrapper = shallow(<Admin {...props}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
