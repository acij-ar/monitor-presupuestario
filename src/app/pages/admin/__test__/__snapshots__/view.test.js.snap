// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin page should match snapshot 1`] = `
<Page
  pageName="admin"
>
  <div
    className="monitor-highlight"
  >
    <div
      className="monitor-content"
    >
      <h1>
        Admin
      </h1>
    </div>
  </div>
  <Component />
</Page>
`;
